src/DataLayer/KeyMotorData.cpp

```diff
@@ -0,0 +1,142 @@
+#include "KeyMotorData.h"
+
+KeyMotorData::KeyMotorData()
+    : motor0Alive_(true)
+    , motor0SetCurrent_(1.1f)
+    , motor0SetVelocity_(2.2f)
+    , motor0BusCurrent_(3.3f)
+    , motor0BusVoltage_(4.4f)
+    , motor0VehicleVelocity_(5.5f)
+    , motor1Alive_(false)
+    , motor1SetCurrent_(6.6f)
+    , motor1SetVelocity_(7.7f)
+    , motor1BusCurrent_(8.8f)
+    , motor1BusVoltage_(9.9f)
+    , motor1VehicleVelocity_(10.01f)
+{
```

Comment:
Can someone tell me why the above code isn't in the bracket? It looks as though it should be

Reply:
It's an initialization list